src/AbstractLoader.php

```diff
@@ -45,21 +50,24 @@ public function load($config, array $options = array())
     /**
      * Add an include alias to the loader
      *
-     * @param string $filename Filename to alias (e.g. _foo)
-     * @param string $alias    Actual file to use (e.g. /path/to/foo.json)
+     * @param string $alias
+     *   Filename to alias (e.g. _foo).
+     * @param string $filename
+     *   Actual file to use (e.g. /path/to/foo.json).
```

Comment:
It looks like this can be a relative or absolute path, can this description be expanded to note that.